api/src/test/java/ai/djl/nn/convolutional/Conv1dTest.java

```diff
@@ -40,7 +40,16 @@ public void testOutputShapes() {
                         * strideWidthRange.size()
                         * dilationWidthRange.size();
 
-        IntStream.range(0, (int) rows)
+        IntStream streamToTest;
+        if (Boolean.getBoolean("nightly")) {
+            // During nightly testing, test all rows
+            streamToTest = IntStream.range(0, (int) rows);
+        } else {
```

Comment:
What are the chances that unit will pass and nightly will fail ? 
Do we have alarm on nightly tests?

Reply:
I would say the chances should be pretty low. Most of the complexity is on the engine side, so we just need to call it appropriately. It has been a standard practice of ours to move the slower tests to nightly to keep the unit test suite fast.

We don't have alarms on nightly, but we probably should. However, that is a problem on its own and I think should be handled separately from this. We already have many nightly only or nightly expanded tests, so this one doesn't change it much.